docs/es-overview.asciidoc

```diff
@@ -121,3 +121,31 @@ to better analyze, visualize, and correlate the data represented in their
 events.
 
 Elastic Security can ingest and normalize events from any ECS-compliant data source.
+
+[discrete]
+[[malware]]
+== Malware prevention
+
+Malware, short for malicious software, is any software program designed to damage or execute unauthorized actions on a
+computer system. Examples of malware include viruses, worms, Trojan horses, adware, scareware, and spyware. Some
+malware, such as viruses, can severely damage a computer's hard drive by deleting files or directory information. Other
+malware, such as spyware, can obtain user data without their knowledge.
+
+Malware may be stealthy and appear as legitimate executable code, scripts, active content, and other software. It is also
+often embedded in non-malicious files, non-suspicious websites, and standard programs — sometimes making the root
+source difficult to identify. If infected and not resolved promptly, malware can cause irreparable damage to a computer
+network.
+
+[discrete]
+=== Machine Learning Model
+
+To determine if a file is malicious or benign, a machine learning model looks for static attributes of files (without executing
+the file) that include file structure, layout, and content. This includes information such as file header data, imports, exports,
+section names, and file size. These attributes are extracted from millions of benign and malicious file samples, which then
+are passed to a machine-learning algorithm that distinguishes a benign file from a malicious one. The machine learning
+model is updated as new data is procured and analyzed.
+
+[discrete]
+=== Threshold
+
+A malware threshold determines the action the agent should take if malware is detected. The Elastic Agent uses a recommended threshold level that generates a balanced number of alerts with a low probability of undetected malware. This threshold also minimizes the number of false positive alerts. For information on how to enable malware protection with the Elastic Agent, see <<install-endpoint, Configure and install Elastic Endpoint Integration (beta)>>.
```

Comment:
Just wondering, if we mention a malware threshold, can users change it?

Reply:
You can change whether to "detect" or "protect" but I don't think you can change the threshold. Adding detect vs prevent instructions in my next commit.